content/installation/setup/installation/UsingZooKeeperProperties.md

```diff
@@ -0,0 +1,55 @@
+<!--
+title: "Configuring Contrast To Use ZooKeeper"
+description: "Instructions for configuring TeamServer to use ZooKeeper as the single source of truth"
+tags: "installation EOP distributed configuration ZooKeeper"
+-->
+
+## About ZooKeeper and Contrast
+This guide is for enterprise-on-premises (EOP) administrators who are looking to move to a distributed setup of Contrast and want to use ZooKeeper as their configuration management. 
+
+> **Note:** While the following steps guide you through the setup and configuration of additional software, keep in mind that you're responsible for the monitoring, backups and durability of ZooKeeper. 
+
+All code that's shared here is also available in our GitHub [repo](https://github.com/Contrast-Security-OSS/docs), so check back often for updates. You can also submit a Pull Request if you have suggestions or find any bugs in the instructions.
```

Comment:
Having this link seems unnecessary to me - we're just linking them back to the same thing they are currently reading. We already have a link to the repo in the top nav of OpenDocs so don't think we need to link again here in an individual article.